index.html

```diff
@@ -1,16 +1,269 @@
+    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
+    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
+    <label class="md-overlay" for="__drawer"></label>
+    <div data-md-component="announce">
+
+    </div>
+      <div data-md-component="outdated" hidden>
+        <aside class="md-banner md-banner--warning">
+        </aside>
+      </div>
+
+<header class="md-header" data-md-component="header">
+  <nav class="md-header__inner md-grid" aria-label="Header">
+    <div class="md-header__title" data-md-component="header-title" width="100%">
+      <div align="center">
+        <span class="md-ellipsis">
+          Eclipse iceoryx™ is an inter-process-communication middleware that enables virtually limitless data transmissions at constant time
+        </span>
+      </div>
+    </div>
+  </nav>
+</header>
+
+    <div class="md-container" data-md-component="container">
+      <main class="md-main" data-md-component="main">
+        <div class="md-main__inner md-grid" width="100%">
+          <div class="md-content" data-md-component="content"  align="center">
+            <article class="md-content__inner md-typeset" style="max-width: 1000px">
+
+<p align="center">
+    <img src="https://user-images.githubusercontent.com/8661268/114321508-64a6b000-9b1b-11eb-95ef-b84c91387cff.png" width="50%">
+
+
+<table align="center">
+<thead>
+<tr>
+<th><h2>iceoryx</h2></th>
+<th><h2>iceoryx2</h2></th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td width="50%">
+  <ul>
+    <li>C++</li>
+    <li>True zero-copy</li>
+    <li>Central daemon</li>
+    <li>Multiple messaging patterns
+      <ul>
+        <li>publish-subscribe</li>
+        <li>request-response</li>
+      </ul>
+    </li>
+    <li>Events</li>
+    <li>Wide range of platforms
+      <ul>
+        <li>Linux</li>
+        <li>Windows</li>
+        <li>macOS</li>
+        <li>FreeBSD</li>
+        <li>QNX</li>
+        <li>FreeRTOS</li>
+      </ul>
+    </li>
+    <li>Some language bindings
+      <ul>
+        <li>C</li>
+        <li>Rust (partial)</li>
+      </ul>
+    </li>
+    <li>Host2Host communication via DDS</li>
+  </ul>
+</td>
+<td>
+  <ul>
+    <li>Rust</li>
+    <li>True zero-copy</li>
+    <li>No central daemon</li>
+    <li>Multiple messaging patterns
+      <ul>
+        <li>publish-subscribe</li>
+        <li>request-response</li>
+        <li>pipeline (planned)</li>
+        <li>blackboard (planned)</li>
+      </ul>
+    </li>
+    <li>Events</li>
+    <li>Wide range of platforms
+      <ul>
+        <li>Linux</li>
+        <li>Windows</li>
+        <li>macOS</li>
+        <li>FreeBSD</li>
+        <li>Android (planned)</li>
+        <li>QNX (planned)</li>
+        <li>FreeRTOS (planned)</li>
+        <li>VxWorks (planned)</li>
+      </ul>
+    </li>
+    <li>Wide range of language bindings
```

Comment:
I think adding this section called as below once they are available is a better idea.

```suggestion
    <li>Available language bindings
```

I understand that you want to communicate to the reader that iceoryx2 is better and more future proven. Instead of using "(planned)" I would actually make this a separate bullet point stating something along the lines of "iceoryx2 is future proven due to memory safe language and its architecture".

Reply:
How about just "Language bindings" in both cases?

Like mentioned above, I think it's valuable to have the planned features listed

Reply:
"Language bindings" is ok. While you are right in saying that the planned features are important, I think it might make sense to link to them from one bullet point "Many features like foo planned, see here" and provide a more detailed description of the planned features e.g. https://github.com/eclipse-iceoryx/iceoryx2/blob/main/ROADMAP.md

Reply:
I updated the comparison. See new screenshots

Reply:
@elBoberido This looks great and is easily digestible. Thanks!